Doc/using/configure.rst

```diff
@@ -289,6 +289,143 @@ General Options
 
    .. versionadded:: 3.13
 
+.. cmdoption:: PKG_CONFIG
+
+   Path to ``pkg-config`` utility.
+
+.. cmdoption:: PKG_CONFIG_LIBDIR
+.. cmdoption:: PKG_CONFIG_PATH
+
+   ``pkg-config`` options.
+
+
+C Compiler Options
```

Comment:
Nitpick:

```suggestion
C compiler options
```

Reply:
Hm, I see the rest of this file is deviating from the devguide recommendation for capitalisation. Perhaps we'd better stay with the current style.

Reply:
I wrote the first version of this doc, and I have the bad habit of using French conventions. Instead, I prefer to capitalize all existing titles: I updated my PR for that.

Reply:
> Perhaps we'd better stay with the current style.

Ah, the prevalent style _is_ sentence case :) So, the PR should be updated to use sentence case.

Reply:
The [devguide](https://devguide.python.org/documentation/style-guide/#capitalization) says:

> In the Python documentation, the use of sentence case in section titles is preferable, but consistency within a unit is more important than following this rule. If you add a section to a chapter where most sections are in title case, you can either convert all titles to sentence case or use the dominant style in the new section title.

So: 

* Either use the prevalent style (sentence case here), or convert to sentence case. 
* Don't convert others to uppercase.